tests/functional/utils/DefaultResponses.h

```diff
@@ -81,6 +81,64 @@ class DefaultResponses {
     partitions.SetPartitions(partitions_vect);
     return partitions;
   }
+
+  static std::string GenerateQuadTreeResponse(
```

Comment:
Here you should generate valid AggregatedDataResult. The same result than should be received from mock server.

Reply:
I don't get it :-)
This is response for `QueryApi::QuadTreeIndex()`, not `AggregatedDataResult()`, which is json with quad tree.

Reply:
There are no serialize\parse stuff for QuadTreeIndex, so can't return model as result.
Moved the method to test.